R/communities.R

```diff
@@ -76,9 +75,33 @@ find_partition <- function(graph, edge_weights, resolution=1.0, niter = 2.0) {
   find_partition_rcpp(edgelist, edgelist_length, num_vertices, direction, edge_weights, resolution, niter)
 }
 
+#' Finds the optimal partition using the Leiden algorithm with replicate starts
```

Comment:
This looks nice. Could you add one more sentence for an explanation? The GitHub issue + PR discussion has been very informative